packages/flutter/test/gestures/debug_test.dart

```diff
@@ -145,4 +145,14 @@ void main() {
     debugPrintRecognizerCallbacksTrace = false;
     debugPrint = oldCallback;
   });
+
+  test('TapGestureRecognizer _sentTapDown toString', () {
+    final TapGestureRecognizer tap = new TapGestureRecognizer();
+    expect('TapGestureRecognizer#fa6ed(state: ready)', equalsIgnoringHashCodes(tap.toString()));
+    var event = const PointerDownEvent(pointer: 1, position: const Offset(10.0, 10.0));
+    tap.addPointer(event);
+    tap.didExceedDeadline();
+    expect(tap.toString(),
+      equalsIgnoringHashCodes('TapGestureRecognizer#fa6ed(state: possible, sent tap down)'));
```

Comment:
make the hashcode in the string be #00000

Reply:
(you can probably just make this one line, you don't save much by wrapping here)